seed/seed.json

```diff
@@ -579,12 +579,20 @@
             "name": "AdServingStudy",
             "experiments": [
                 {
-                    "name": "MaximumAdNotificationsPerDay=100",
+                    "name": "MaximumAdNotificationsPerDay=100/MaximumInlineContentAdsPerHour=8/MaximumInlineContentAdsPerDay=40",
```

Comment:
not sure this is convenient, mb use shorter names?..

Reply:
We have been using the same name as the related params